x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/EqlSearchResponse.java

```diff
@@ -323,9 +362,13 @@ public Map<String, DocumentField> fetchFields() {
             return fetchFields;
         }
 
+        public boolean missing() {
+            return Boolean.TRUE.equals(missing);
```

Comment:
Yes, it's just about the c'tor param (it has to be an Object, because it's optional - x-content bwc, you know), but it's probably better to have the attribute to be a `boolean`, I'll refactor it